templates/python/metadata_tags_metrics.patch

```diff
@@ -1,74 +1,77 @@
 diff --git a/python/docs/ModelMetadataServiceApi.md b/python/docs/ModelMetadataServiceApi.md
-index 22abe99..6fe61b7 100644
+index 8375e1a..0eede35 100644
```

Comment:
Nice one on fixing the patch! It has been giving us troubles for a while!

Reply:
It was a sad, manual process, but hopefully it won't need to be redone too often.
If it does, I can add instructions on how I did it. (Although there is probably a better way!)